styles.css

```diff
@@ -2,6 +2,21 @@ body {
   background-color: #78a1bb;
 }
 
-.car {
+.car,
+.north,
+.south,
+.west {
   width: 10%;
 }
```

Comment:
Once you stop comma separating classes, you can target it using the `.car` class.